src/components/ApplicationListView/ApplicationListView.tsx

```diff
@@ -36,13 +40,40 @@ const ApplicationListView: React.FC<React.PropsWithChildren<unknown>> = () => {
   const applicationBreadcrumbs = useApplicationBreadcrumbs();
   const [canCreateApplication] = useAccessReviewForModel(ApplicationModel, 'create');
   const [canCreateComponent] = useAccessReviewForModel(ComponentModel, 'create');
+  const [nameFilter, setNameFilter] = useSearchParam('name', '');
+  const [onLoadName, setOnLoadName] = React.useState(nameFilter);
+
+  React.useEffect(() => {
```

Comment:
Why do we need a `nameFilter` and also an `onLoadName` , in this hook, the `onLoadName` keeps updating with the `nameFilter` doesn't it?

Reply:
Seems redundant, we can only use the query params from namefilter. Removed it.